### PR TITLE
Minor improvements

### LINKED ISSUE #28: Yaru: tiny improvements
**Version Info**
- Theme: Yaru (not dark)

**Description**
1. the badges are orange, but in yaru they are green ```#0e8420``` 
2. The terminal is correctly purple as in yaru but in a super white window, this hardcore contrast to the dark purple is imho too much therefor I would suggest ```#f7f7f7``` for the background and ```#3b3b3b``` for the foreground
3. The selection color is dark purple with a dark foreground in the light yaru version. The selected text is unreadable, I did not find the correct color for the foreground color but I would either suggest to somehow transparentize the purple or to have a white selection text color :) 
grafik

**Screenshots**
Screenshot of the suggestion:
Bildschirmfoto von 2020-07-14 13-52-25

Thanks for bringing yaru to vscode and best regards :beer:

### PR DESCRIPTION
**Pull request by Feichtmeier**
- use orginal yaru green badges
grafik

~~use a light terminal to reduce the heavy contrast against the bright window~~

Closes #28

### CODE REVIEW

File: themes/dark/yaru-dark.json [hunk 1]
```diff
@@ -356,9 +356,9 @@
     "list.inactiveSelectionBackground": "#575757",
     "activityBar.background": "#575757",
```
Comment (rdnlsmith) [inline themes/dark/yaru-dark.json]: I'm thinking that `activityBar.background` and `input.background` should be darkened to `#484848` to improve contrast against the new badge color. I got `#484848` from the Software Center.

badge-yaru-dark-contrast

File: themes/dark/yaru-dark.json [hunk 2]
```diff
@@ -356,9 +356,9 @@
...
     "activityBar.dropBackground": "#e9542080",
-    "activityBarBadge.background": "#e95420",
+    "activityBarBadge.background": "#0c6d1a",
     "activityBarBadge.foreground": "#ffffff",
-    "badge.background": "#e95420",
+    "badge.background": "#991121",
```
Comment (rdnlsmith) [inline themes/dark/yaru-dark.json:361]: This looks good, but I'm curious about the rationale? I don't think I've seen badges using this color while using Ubuntu.

**Review by rdnlsmith — CHANGES_REQUESTED** [review]
I like the new colors; I mentioned in another comment that I'd like to darken the activity bar of Yaru Dark for better contrast with the dark green badges.

I mentioned in the issue that the Yaru JSON files are generated from some TypeScript files. Would you mind moving your changes to `themes/src/light/yaru.ts` and `themes/src/dark/yaru-dark.ts`? Basically you would use the same key name but put it in the `overrides` section. One nice thing about this is you can also give names (_e.g._ `activityBarGray`, `badgeGreen`, _etc._) to the hex colors by adding them to the `colors` section.

To preview, (run `npm install` and also have `npx` installed, and then) run `npm run start`. Whenever you save one of the `.ts` files it will automatically re-generate, so you still get a live preview albeit with a bit of a delay. Let me know if you have any trouble with this&mdash;I don't believe anyone but me has attempted to run this since I set it up, and I only _kind of_ know what I'm doing when it comes to node.

File: themes/dark/yaru-dark.json (hunk 2 above)
Comment (Feichtmeier) [reply themes/dark/yaru-dark.json:361]: This change is more to the nature of that green in yaru. We use it as a "success" color, i.e. for new things that are not negative. That badges in the bottom panel are generally more of "attention!" things, mostly errors or warnings. So iff they are green it is the wrong message: success, you have an error :smile: 
Good Idea with the contrast, I'll change that for the dark theme!

Comment (rdnlsmith) [reply themes/dark/yaru-dark.json:361]: Ah, that definitely makes sense for the problems pane! They also show up on the search results pane, which is why I asked, but I think it's fine.

**Comment by rdnlsmith** [comment]
@Feichtmeier Thanks!

I will likely publish an update in the next couple days. I usually tweet about it&mdash;would you like to be tagged in the tweet for attribution, and if so, what's your Twitter username?

**Comment by Feichtmeier** [comment]
@rdnlsmith no twitter handle currently :) Thanks for the merge! :beer:

**Pull request merged**